libglusterfs/src/iobuf.c

```diff
@@ -509,6 +509,37 @@ iobuf_get_from_stdalloc(struct iobuf_pool *iobuf_pool, const size_t page_size)
     return iobuf;
 }
 
+static struct iobuf *
+iobuf_get_from_small(struct iobuf_pool *iobuf_pool, const size_t page_size)
+{
+    struct iobuf *iobuf = NULL;
+    int ret = -1;
+
+    iobuf = GF_CALLOC(1, sizeof(*iobuf), gf_common_mt_iobuf);
```

Comment:
Why not use `GF_MALLOC()` ?

Reply:
It was crashing so i use GF_CALLOC.

Reply:
If all fields are initialized it cannot crash.

Reply:
done